shared/cfg/guest-hw.cfg

```diff
@@ -82,23 +75,6 @@ variants:
         # Add -drive ...boot=yes unless qemu-kvm is 0.12.1.2 or newer
         # then kvm_vm will ignore this option.
         image_boot=yes
-        pseries:
```

Comment:
The `pseries` branch is defined in `machines.cfg` so this change should not affect it. What this change does is it changes the driver inside the `qemu_vm` rather than in `cfg` files, which works nice for both, qemu and libvirt provider as libvirt provider.

I listed the unattended_install cmdlines and you can see it hasn't changed for ppc, nor for the other providers.